crowbar_framework/app/models/node_object.rb

```diff
@@ -1207,7 +1207,7 @@ def run_ssh_cmd(cmd)
     args = ["sudo", "-i", "-u", "root", "--", "timeout", "-k", "5s", "15s",
             "ssh", "-o", "ConnectTimeout=10",
             "root@#{@node.name}",
-            cmd.to_s
+            "\"#{cmd}\""
```

Comment:
Ah, the commit is https://github.com/crowbar/crowbar-core/pull/623/commits/3bb7e0cec091a823262543531c7d4a23aee5c682 now. (Seems like I fixed up last change incorrectly)

> I guess only root has the required authorized key?

That seems correct. I just based the method on existing `ssh_cmd`